### DISCUSSION TITLE
Is ModelSchema Config class replacement for the Meta class?

### DISCUSSION THREAD

**Discussion by EmilTholin** [vitalik/django-ninja#1014] [Q&A] [answered]
Hi! I'm a big fan of what you have accomplished with Django Ninja, I will use it in my next project.

I was studying the documentation and I came across this part [`Schemas from Django models`](https://django-ninja.dev/guides/response/django-pydantic/), which states that:

> All you need is to set `model` and `fields` attributes on your schema `Meta`
```python
from django.contrib.auth.models import User
from ninja import ModelSchema

class UserSchema(ModelSchema):
    class Meta:
        model = User
        fields = ['id', 'username', 'first_name', 'last_name']
```

However, when trying this out you get an error saying that the `Config` class is missing. Changing from `Meta` to `Config`, and from `fields` to `model_fields` does indeed work.

Is this an outdated part of the documentation, or is there some nuance of the `Meta` class I'm missing?

**Comment by vitalik** [comment, marked as answer]
Hi @EmilTholin 

Could you share your ninja version ? (note Meta is available since 1.0, Config deprecated)

**Reply by EmilTholin** [reply]
Hi @vitalik. Thank you for your quick reply.

I'm trying out `0.22.2`, that explains it. I got it twisted somehow, and thought `Config` was the new way to go about it. It does indeed work with `1.1.0`.
